apps/nextjs-website/src/_contents/send/quickStartGuide.ts

```diff
@@ -0,0 +1,462 @@
+import { QuickStartGuideData } from '@/lib/types/quickStartGuideData';
+import { send } from '@/_contents/send/send';
+import { sendBannerLinks } from '@/_contents/send/bannerLinks';
+import { sendQuickStartGuidePath } from '@/_contents/send/quickStartGuidePath';
+
+export const sendQuickStartGuide: QuickStartGuideData = {
+  ...sendQuickStartGuidePath,
+  product: send,
+  abstract: {
+    title: 'Quick start',
+    description:
+      'Se vuoi sapere cosa devi fare per integrarti, sei nel posto giusto. Con la quick start scoprirai in poco tempo tutti i passaggi che compongono il processo di integrazione.',
+  },
+  defaultStepAnchor: '01',
+  steps: [
+    {
+      title: "Genera l'API Key",
+      anchor: '01',
+      parts: [
+        {
+          component: 'innerHTML',
+          html: "Accedi alla <a href='https://dev.selfcare.pagopa.it/auth/login'>piattaforma Self Care</a> ed entra su SEND. In questo video troverai tutti i passaggi per generare le API Key, che <b>dovrà essere utilizzata in tutte le chiamate API</b>.",
```

Comment:
**Richiesta di sostituzione testo:**
"In questo video troverai" --> "In questo video interattivo troverai"